cfme/tests/physical_infrastructure/ui/test_physical_switch_list.py

```diff
@@ -18,7 +18,7 @@ def test_physical_switches_view_displayed(appliance):
     assert physical_switches_view.is_displayed
 
 
-def test_physical_switches_view_dropdowns(appliance):
+def test_physical_switches_view_dropdowns(appliance, physical_switch):
```

Comment:
Is the reason you need `physical_switch` the same as above -- i.e. the menu item is disabled without the provider configured?

Reply:
Yep - Including physical_switch here adds the dependency on `provider` + `setup_provider`, which will prevent the test from getting collected if there is no provider in the yamls.  The dropdowns are disabled when there aren't entities present.

If there is a provider in the yamls, but that provider doesn't actually have a switch available, then the test will skip in setup via the fixture instead of raising an exception.